### PR TITLE
Sliding Animated Form using HTML, CSS and JS

### PR DESCRIPTION
**Pull request by Mridul2820**
## Sliding Animated Login and Sign up Form using HTML, CSS and JS

Screenshot:

image

### CODE REVIEW

File: sliding-form/index.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+                    <p>Enter your personal details and start journey with us</p>
+                    <button class="ghost" id="signUp">Sign Up</button>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <script src="script.js"></script>
+</body>
+</html>
```
Comment (CaioCordeiro) [inline sliding-form/index.html:63]: Please add an empty line at the end of the file.

File: sliding-form/script.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+const signInButton = document.getElementById('signIn');
+const container = document.getElementById('container');
+
+signUpButton.addEventListener('click', () => {
+	container.classList.add("right-panel-active");
+});
+
+signInButton.addEventListener('click', () => {
+	container.classList.remove("right-panel-active");
+});
```
Comment (CaioCordeiro) [inline sliding-form/script.js:11]: Here too.

File: sliding-form/style.scss [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+    transform: translateX(50%);
+}
+
+.container.right-panel-active .overlay-left {
+    transform: translateX(0);
+}
+
+.container.right-panel-active .overlay-right {
+    transform: translateX(20%);
+}
```
Comment (CaioCordeiro) [inline sliding-form/style.scss:214]: Also here.

**Review by CaioCordeiro — CHANGES_REQUESTED** [review]
What a great code 😄 !
Just a little tip, it's good always to put a description on your PRs to make it more understandable. Could you add it for me?
Thank you for contributing! 🚀

**Comment by CaioCordeiro** [comment]
@Mridul2820  Please add a description to your PR 😄 .
The code looks fantastic.

**Comment by Mridul2820** [comment]
Hi @CaioCordeiro I have added the description and also a screenshot of the page

**Comment by Mridul2820** [comment]
Hi @CaioCordeiro I have added all the changes you asked

**Pull request merged**